resources.md

```diff
@@ -55,6 +55,7 @@ Some of these platforms also take submissions, so if you designed a font with an
 * [New York Public Library Public Domain
 Images](http://www.nypl.org/research/collections/digital-collections/public-domain?hspace=331354)
 * [unDraw](https://undraw.co/)
+* [Interfacer](https://interfacer.xyz/)
```

Comment:
Just looked through a bunch of resources from interfacer and most of them didn’t have clear licenses on where to use it. We should try and focus on not a _mass_ of resources, but about the best ones which have clearly stated open licenses like Creative Commons (CC0, Attribution, Share-Alike), or MIT, BSD, AGPL and the like.